apps/src/music/appConfig.js

```diff
@@ -1,6 +1,8 @@
 import {queryParams} from '@cdo/apps/code-studio/utils';
 import {BlockMode} from './constants';
 
+let setValues = null;
```

Comment:
nit: `setValues` to me sounds like a method name to set the values. Can we rename to be clear these are the config values?

Reply:
Good point.  Will rename to `configValues`.